### DISCUSSION TITLE
Difference between two namespaces

### DISCUSSION THREAD

**Discussion by msedi** [CommunityToolkit/dotnet#114] [Q&A] [answered]
What is the difference between Microsoft.Toolkit.HighPerformance and CommunityToolkit.HighPerformance.
Especially the HighPerformance is much harder to find than the WindowsCommunityToolkit

What is the reason for this?

It seems to me that the toolkits are a bit spreaded. Wouldn't it make sense to put them into one place?

**Comment by Sergio0694** [comment, marked as answer]
They're effectively the same, as of the last public release, which is 7.1.2. Going forward, only CommunityToolkit.* will be updated, as we've changed our name and package identity from Microsoft.Toolkit.* to CommunityToolkit.*, as mentioned in [this blog post](https://devblogs.microsoft.com/ifdef-windows/welcome-to-the-community-toolkit/) 🙂

We'll eventually deprecate the Microsoft.Toolkit.* packages and redirect to the new ones on NuGet.

Note: the packages are still supported and maintained by Microsoft, we just changed the name for several reasons 😄

Also, note that the Windows Community Toolkit and the .NET Community Toolkit are now separate projects. We're working on setting up the new docs with all the updated info, which should help clarify how all the various components of the Toolkit fit together.

Hope this helps!

**Reply by msedi** [reply]
@Sergio0694: Thanks for the prompt info. I was just curious and I honestly missed your blog post ;-)
